cpp/src/Ice/LoggerAdminI.cpp

```diff
@@ -736,11 +737,12 @@ LoggerAdminLoggerI::run()
 
     for(;;)
     {
-        IceUtil::Monitor<IceUtil::Mutex>::Lock lock(_monitor);
-        while(!_destroyed && _jobQueue.empty())
+        unique_lock lock(_mutex);
+        while (!_destroyed && _jobQueue.empty())
         {
-            _monitor.wait();
+            _conditionVariable.wait(lock);
```

Comment:
Should we update this kind of use and use (2)?
https://en.cppreference.com/w/cpp/thread/condition_variable/wait